tests/foreman/api/test_foremantask.py

```diff
@@ -18,37 +18,33 @@
 from nailgun import entities
 from requests.exceptions import HTTPError
 
-from robottelo.test import APITestCase
 
+@pytest.mark.tier1
+def test_negative_fetch_non_existent_task():
+    """Fetch a non-existent task.
 
-class ForemanTaskTestCase(APITestCase):
-    """Tests for the ``foreman_tasks/api/v2/tasks/:id`` path."""
+    :id: a2a81ca2-63c4-47f5-9314-5852f5e2617f
 
-    @pytest.mark.tier1
-    def test_negative_fetch_non_existent_task(self):
-        """Fetch a non-existent task.
+    :expectedresults: An HTTP 4XX or 5XX message is returned.
 
-        :id: a2a81ca2-63c4-47f5-9314-5852f5e2617f
+    :CaseImportance: Critical
```

Comment:
is this actually critical?

Reply:
I'll leave that for the component owner to decide.

Reply:
Agreed @tpapaioa, but it looks like you've changed this test case from default for the module (High) to Critical.